tests/engine/render/material/albedo/test.js

```diff
@@ -24,6 +24,13 @@ autoTester.perform("Effects of albedo on various materials", Script.resolvePath(
         createdEntities = addCases(TEST_CASES, true, true, autoTester.getOriginFrame());
         validationCamera_translate(OFFSET);
     });
+
+    var fxaaWasOn;
+    autoTester.addStep("Turn off TAA for this test", function () {
+        fxaaWasOn = Render.getConfig("RenderMainView.Antialiasing").fxaaOnOff;
+        Render.getConfig("RenderMainView.JitterCam").none();
+        Render.getConfig("RenderMainView.Antialiasing").fxaaOnOff = true;
+    });
```

Comment:
Seems like we should add this to the autotester somehow so we don't have to copy it everywhere?

Reply:
I was thinking the same.  I was hesitant to do this at the beginning, but it seems that TAA should be disabled for all tests (bar one...)
I will do that.

Reply:
Tests have been modified - TAA disabling now part of autoTester.js